lang/en.json

```
@@ -498,6 +498,15 @@
 "ADVANCEMENT.MilestoneAward": "Award Milestones",
 "ADVANCEMENT.ExpectedLevel": "Expected Level",
 
+"AWARD": {
+  "InvalidWarning": "Unable to parse {terms} as valid award terms.",
+  "Milestone": {
+    "one": "Milestone",
+    "other": "Milestones"
+  },
+  "NotGMWarning": "You must be a Gamemaster to use Award enrichers."
```

Let's anticipate likely additional warnings as we continue to develop this feature and go with:
```suggestion
  "WARNINGS": {
    "RequiresGM": "You must be a Gamemaster to use Award enrichers."
  }
```